app/scripts/modules/core/delivery/executionGroup/execution/Execution.tsx

```diff
@@ -267,6 +262,7 @@ export class Execution extends React.Component<IExecutionProps, IExecutionState>
   }
 
   public render() {
+    const { AccountTag, ExecutionStatus, PipelineGraph, ExecutionDetails, CopyToClipboard } = ReactInjector;
```

Comment:
I think this reads a bit better in JSX than the other way `<ReactInjector.SomeComponent/>`